src/js/components/home/toolsManagement/ToolsCards.js

```diff
@@ -5,72 +5,72 @@ import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
-          <Card style={{ display: "flex", justifyContent: "center", alignItems: "center", margin: "6px 0px 10px", height: "200px" }}>
-            <CardText style={{ fontWeight: "bold" }}>
-              No tC default tools found.
-            </CardText>
-          </Card>
-        </MuiThemeProvider>
-      );
-    } else if (bookName.length == 0 && projectSaveLocation == 0) {
-      return (
-        <MuiThemeProvider>
-          <Card style={{ display: "flex", justifyContent: "center", alignItems: "center", margin: "6px 0px 10px", height: "200px" }}>
-            <CardText style={{ fontWeight: "bold" }}>
-              No project was selected. Please
-              <span
-                style={{ color: "var(--accent-color-dark)", cursor: "pointer" }}
-                onClick={() => this.props.actions.goToStep(2)}
-              >
-                &nbsp;select a project&nbsp;
-              </span>first.
-            </CardText>
-          </Card>
-        </MuiThemeProvider>
-      );
-    } else {
-      return (
-        <div style={{ height: '100%', overflowY: 'auto', paddingRight: '10px' }}>
-          {
-            toolsMetadata.map((metadata, i) => {
-              if (metadata.title === "translationWords (Part 1)") {
-                return (
-                  <ToolCard
-                    key={i}
-                    actions={this.props.actions}
-                    loggedInUser={loggedInUser}
-                    metadata={metadata}
-                    currentProjectToolsProgress={currentProjectToolsProgress}
-                  />
-                );
-              }
-            })
-          }
-        </div>
-      );
-    }
+  if (toolsMetadata.length == 0 || !toolsMetadata) {
+    return (
+      <MuiThemeProvider>
+        <Card style={{ display: "flex", justifyContent: "center", alignItems: "center", margin: "6px 0px 10px", height: "200px" }}>
+          <CardText style={{ fontWeight: "bold" }}>
+            No tC default tools found.
+          </CardText>
+        </Card>
+      </MuiThemeProvider>
+    );
+  } else if (bookName.length == 0 && projectSaveLocation == 0) {
+    return (
+      <MuiThemeProvider>
+        <Card style={{ display: "flex", justifyContent: "center", alignItems: "center", margin: "6px 0px 10px", height: "200px" }}>
+          <CardText style={{ fontWeight: "bold" }}>
+            No project was selected. Please
+            <span
+              style={{ color: "var(--accent-color-dark)", cursor: "pointer" }}
+              onClick={() => this.props.actions.goToStep(2)}
+            >
+              &nbsp;select a project&nbsp;
+            </span>first.
+          </CardText>
+        </Card>
+      </MuiThemeProvider>
+    );
+  } else {
+    return (
+      <div style={{ height: '100%', overflowY: 'auto', paddingRight: '10px' }}>
+        {
+          toolsMetadata.map((metadata, i) => {
+            if (developerMode ? developerMode : metadata.title === "translationWords (Part 1)") {
```

Comment:
the title is the best metadata we can use since its the only one that specifies if it is `part 1`